tests/test-unattended-config.c

```diff
@@ -249,15 +249,15 @@ test_missing_product (void)
 }
 
 static void
-test_full_blockdevice_path (void)
+test_full_block_devicepath (void)
```

Comment:
s/`devicepath`/`device_path`/, or was that intentional?

Reply:
It was not.